src/cli/commands/list.js

```diff
@@ -180,21 +181,33 @@ export function filterTree(tree: Tree, filters: Array<string>): boolean {
   return notDim && (found || hasChildren);
 }
 
+const getDevDeps = (manifest): Array<string> => Object.keys(manifest.devDependencies).map(key => `${key}@${manifest.devDependencies[key]}`);
+
 export async function run(config: Config, reporter: Reporter, flags: Object, args: Array<string>): Promise<void> {
   const lockfile = await Lockfile.fromDirectory(config.lockfileFolder, reporter);
   const install = new Install(flags, config, reporter, lockfile);
-  const {requests: depRequests, patterns, workspaceLayout} = await install.fetchRequestFromCwd();
+
+
+  const {requests: depRequests, patterns, manifest, workspaceLayout} = await install.fetchRequestFromCwd();
   await install.resolver.init(depRequests, {
     isFlat: install.flags.flat,
     isFrozen: install.flags.frozenLockfile,
     workspaceLayout,
   });
 
+  let pkgPatterns = [];
+  if (isProduction(process.env)) {
```

Comment:
I would just use `process.env` at all times instead of passing this. Also this should probably be a method on `config` since we also support passing `--prod` flag. Not sure that flag applies to `list` but we can make it so?